44 46 UI Details

jeffbuonamici opened the pull request:
#44 #46

- Created controllers for each scene
- Created CSS files for each scene
- Implemented the necessary JavaFX elements to display the data.
- Created Engine and Sensor classes to hold data in a usable format for the front end.
- Created the onClick events to link between scenes.

jeffbuonamici committed:
#46 #44 Created Controller, Engine and Sensor classes

Created controllers needed for the FXML pages
Created Engine and Sensor objects
Added CSS files for FXML pages
Editted some FXML pages.

jeffbuonamici committed:
#44 Generating Sensor Panes

jeffbuonamici committed:
#46 #66 Created Controller for Add System page.

Created Controller for Add System page
Fixed some styling
Linked pages to one another

jeffbuonamici committed:
#44 #46 Refactoring and comments

PaulMicu reviewed (changes requested):
please remove target files from pull request

rdsaliba commented on src/main/java/com/cbms/app/Engine.java:
```diff
@@ -0,0 +1,101 @@
+package com.cbms.app;
+
+public class Engine {
```
We created another model for this called "Asset"

rdsaliba reviewed (commented):
I believe you must switch your engine class to our asset class to match the mimosa standard. Let me know what you think.

PaulMicu committed:
DB Queries for Instances extraction

Connection to db

PaulMicu committed:
added DB scripts

PaulMicu committed:
instances created and modles trained

PaulMicu committed:
code cleanup

jeffbuonamici committed:
Bug fixes

jeffbuonamici committed:
Merge

jeffbuonamici committed:
Leftover Conflict

jeffbuonamici committed:
#44 #46  Changed UI to work with Asset and AssetAttribute classes

jeffbuonamici committed:
Delete lib.xml

jeffbuonamici committed:
Delete inputFiles.lst

jeffbuonamici committed:
Delete systems.css

jeffbuonamici committed:
Delete systeminfo.css

jeffbuonamici committed:
Delete style.css

jeffbuonamici committed:
Delete Systems.fxml

jeffbuonamici committed:
Delete SystemInfo.fxml

jeffbuonamici committed:
Delete AddSystem.fxml

jeffbuonamici commented:
I believe I fixed your requests.
- I'm now using Asset and AssetAttribute objects instead of Engine and Sensor
- Deleted target folder files that were committed

Also, Paul helped me update the branch from the Backend Connections branch to avoid merge errors I believe.

rdsaliba reviewed (approved):
Ready to merge
